dev-tools/create_bwc_index.py

```diff
@@ -315,16 +323,26 @@ def create_bwc_index(cfg, version):
     # this after the snapshot, because it calls flush.  Otherwise the index
     # will already have the deletions applied on upgrade.
     delete_by_query(client, version, index_name, 'doc')
-    
+
+    logging.info('Shutting down node with pid %d', node.pid)
+    node.terminate()
+    node.wait()
+    node = None
+
+    compress_index(version, tmp_dir, cfg.output_dir)
+    if snapshot_supported:
+      compress_repo(version, tmp_dir, cfg.output_dir)
   finally:
-    if 'node' in vars():
+
+    if node is not None:
+      # This only happens if we've hit an exception:
       logging.info('Shutting down node with pid %d', node.pid)
```

Comment:
maybe move this to a simple helper like `shutdown_node(node)` since it is exactly the same as above?